### PR TITLE
Alignment-Aplitop-1

### PR DESCRIPTION
**Pull request by javierNadal**
## Alignment1

*This scenario is a very simple alignment (horizontal + vertical).*

## Covered topics

*tick one/many*

- [x] project setup
- [ ] georeferencing
- [ ] alignment semantics
- [x] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

File: Align_1/UT_Align_1.IFC [hunk 1]
```diff
@@ -0,0 +1,208 @@
+/* Generated by TcpMDT - V8.5
+   by APLITOP, S.L. (www.aplitop.com)
+*/
```
Comment (pjanck) [inline Align_1/UT_Align_1.IFC]: Please use "-" instead of "_" in files' and folders' names.

File: Align_1/readme.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+## *Align_1*
```
Comment (pjanck) [inline Align_1/readme.md]: No need for a title - please delete this line.
```suggestion

```

File: Align_1/readme.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+### Intent
+
+*This scenario is a very simple alignment (horizontal + vertical).*
```
Comment (pjanck) [inline Align_1/readme.md]:
```suggestion
This scenario is a very simple alignment (horizontal + vertical).
```

Please use normal font. The italics is used in the template to denote the content you need to input/change.

File: Align_1/readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+### Prerequisites
+
+No prerequisites
```
Comment (pjanck) [inline Align_1/readme.md]: I believe that ProjectSetup_1 would be a good base test.

File: Align_1/readme.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+Following files correspond to this scenario:
+
+| Filename                     | Description                               |
+|------------------------------|-------------------------------------------|
+| *UT_Align_1.ifc*             | *Exported data as IFC*             |
+| *UT_Align_1.dwg*             | *CAD file with horizontal and vertical alignments representation* |
+| *UT_Align_1._horizontal.png* | *screen shot of CAD file*                 |
+| *UT_Align_1_Vertical.png*    | *screen shot of CAD file*                 |
+| *UT_Align_1.xml*             | *same content of Ifc file, in LandXML format* |
```
Comment (pjanck) [inline Align_1/readme.md]: Please use Grave Accent symbol for code snippets (like file names, or IFC entities. Also, no need for italics.

**Review by pjanck — CHANGES_REQUESTED** [review]
Please observe the markdown syntax and requirements for file naming.

**Comment by pjanck** [comment]
Please also merge in the new changes from the `main` branch so that IFC checking procedure runs your files as well.

**Comment by pjanck** [comment]
Following the changes to the schema, (we are now at `RC3`), we ask you to update your IFC files accordingly. The schema can be found [here](https://github.com/bSI-InfraRoom/IFC-Specification/blob/main/IFC4x3_RC3.exp). The checking procedure for IFC files has been updated - please merge in the current state of the `main` branch.

**Comment by javierNadal** [comment]
I don't know how remove IFC4x3_RC2 and add IFC4x3_RC3 labels...

**Comment by AlexBrad1eyCT** [comment]
Done it for you but for reference!

image

**Pull request merged**